config/toml.go

```diff
@@ -23,12 +23,12 @@ log-level = "{{ .BaseConfig.LogLevel }}"
 oracle-mnemonic = "{{ .BaseConfig.OracleMnemonic }}"
 oracle-acc-num = "{{ .BaseConfig.OracleAccNum }}"
 oracle-acc-index = "{{ .BaseConfig.OracleAccIndex }}"
-listen_addr = "{{ .BaseConfig.ListenAddr }}"
-data_dir = "{{ .BaseConfig.DataDir }}"
+listen-addr = "{{ .BaseConfig.ListenAddr }}"
```

Comment:
Does we need to move `listen-addr` in API Configuration or just leave in here?

Reply:
I think `listen-addr` is only used in API. It would be better to move it. Thanks.